guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -139,3 +139,28 @@ include::snip_steps-needs-reboot.adoc[]
 ifdef::orcharhino,satellite[]
 . Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} using remote execution
+
+. Create a backup or take a snapshot.
++
+For more information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
+
+. In the {ProjectWebUI} , navigate to *Monitor* > *Jobs*.
+
```

Comment:
If there is no guideline about procedure step spacing, I recommend removing the empty lines to save space and make it easier for reviewers to scan. (Obviously the rendering will be the same, regardless of the spacing.)

Reply:
Yes, by convention, we do not add unnecessary whitespace (e.g. between list items)